objectivec/GPBUtilities_PackagePrivate.h

```diff
@@ -36,13 +36,22 @@
 
 // Macros for stringifying library symbols. These are used in the generated
 // PB descriptor classes wherever a library symbol name is represented as a
-// string. See README.google for more information.
+// string.
 #define GPBStringify(S) #S
 #define GPBStringifySymbol(S) GPBStringify(S)
 
 #define GPBNSStringify(S) @#S
 #define GPBNSStringifySymbol(S) GPBNSStringify(S)
 
+// Macros for generating a Class from a class name. These are used in
+// the generated PB descriptor classes wherever an Objective C class
```

Comment:
PB -> GPB ?

Reply:
Done (and in the comment above that I probably copied from)